karma.conf.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (config) {
+	config.set({
+		frameworks: ['ava'],
+		files: [
+			'test/shortcut-*.js',
+			'test/zen-provided.js'
+		],
+		browsers: [process.env.CI ? 'PhantomJS' : 'Chrome'],
```

Comment:
Any way Karma can just do this automatically? Seems like a good default.

That being said, can't wait for headless Chrome! https://twitter.com/samccone/status/739166801427210240